browser/importer/brave_profile_writer.cc

```diff
@@ -455,28 +462,16 @@ void BraveProfileWriter::UpdateWindows(
 
 // NOTE: the strings used as keys match the values found in Muon:
 // browser-laptop/js/data/searchProviders.js
+// Providers that aren't in this map are no longer prepopulated (Amazon,
+// Ecosia, GitHub, etc.) and the current default provider won't be changed.
```

Comment:
What happens if someone has set one of the removed engines as their default (ex: amazon)? Does it change back to Google (or whatever the original default is) when they update Brave?

Reply:
nvm, just saw the test plan